controllers/review.js

```diff
@@ -15,10 +15,10 @@ const getReviewById = async (req, res) => {
 }
 
 const addReview = async (req, res) => {
-  const { id: authorId } = req.params
-  const { comment, rating, targetUserId, targetUserRole, offerId } = req.body
+  const { id: author } = req.params
```

Comment:
Why did you change it? It's not obvious now.

Reply:
i did it becasue now there is populated information about author contains in this field in requests and i think this naming fits better